conjure-undertow-lib/src/main/java/com/palantir/conjure/java/undertow/lib/Markers.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.conjure.java.undertow.lib;
+
+import io.undertow.server.HttpServerExchange;
+
+public interface Markers {
```

Comment:
this name isn't good. `MarkerCallback`?

Reply:
MarkerCallback sounds good to me. @carterkozak thoughts?

Reply:
renamed to MarkerCallback